src/osd/PrimaryLogPG.cc

```diff
@@ -11826,6 +11826,11 @@ void PrimaryLogPG::handle_watch_timeout(WatchRef watch)
   oi.watchers.erase(make_pair(watch->get_cookie(),
 			      watch->get_entity()));
 
+  osd->logger->inc(l_osd_watch_timeout);
+  dout(3) << __func__ << " watcher: "
+         << watch->get_peer_addr()
```

Comment:
If introducing a separate dout at level 3 (pretty high), I wonder if logging just the peer address without also at least the object name is useful?

Reply:
1. The reason for using level 3 is because I believe that the code can run to here may be due to a malfunction occurring.
2. you are right, I was just about to add  the object name.